src/CLI/clitree/cli-xml/ifa.xml

```diff
@@ -0,0 +1,94 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<!DOCTYPE CLISH_MODULE [
+]>
+<!--
+Copyright 2019 Dell, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+<CLISH_MODULE
+    xmlns="http://www.dellemc.com/sonic/XMLSchema"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xmlns:xi="http://www.w3.org/2001/XInclude"
+    xsi:schemaLocation="http://www.dellemc.com/sonic/XMLSchema
+                        http://www.dellemc.com/sonic/XMLSchema/clish.xsd"
+    >
+<!--=========================================================================-->
+
+  <VIEW name="enable-view">
+    <COMMAND name="show tam-int-ifa" help="Show TAM IFA info"/>
+    <COMMAND name="show tam-int-ifa supported" help="Show TAM IFA feature info" mode="subcommand" ptype="SUBCOMMAND">
+      <ACTION>python $SONIC_CLI_ROOT/sonic-cli-ifa.py get_sonic_ifa_sonic_ifa_tam_int_ifa_feature_table show_tam_feature.j2</ACTION>
+    </COMMAND>
+    <COMMAND name="show tam-int-ifa flow" help="Show TAM IFA flow info" mode="subcommand" ptype="SUBCOMMAND">
+      <PARAM name="name" help="Name of the collector" ptype="STRING_32" optional="true"> </PARAM>
+      <ACTION>
```

Comment:
"show tam-int-ifa" needs to be changed to "show tam int-ifa". Hence the new commands are "show tam int-ifa supported", "show tam int-ifa flow" etc.

As specified earlier show for tam and show tam int-ifa needs to be in their specific feature files..

Reply:
Done